pkg/vpn_server/values.go

```diff
@@ -21,52 +22,82 @@ func BuildValues(cfg config.VPNServer) (openvpn.SeedServerValues, error) {
 		StatusPath: cfg.StatusPath,
 	}
 
-	v.ShootNetworks = append(v.ShootNetworks, cfg.ServiceNetworks...)
-
-	v.ShootNetworks = append(v.ShootNetworks, cfg.PodNetworks...)
-
-	if len(cfg.NodeNetworks) != 0 && cfg.NodeNetworks[0].String() != "" {
-		v.ShootNetworks = append(v.ShootNetworks, cfg.NodeNetworks...)
+	if len(cfg.VPNNetwork.IP) != 16 {
+		return v, fmt.Errorf("VPN_NETWORK must be a IPv6 CIDR: %s", cfg.VPNNetwork)
```

Comment:
This check can be true for an IPv4 address in the IPv6 representation, e.g. `net.IPv4(1,2,3,4)` returns an IPv4 address of length 16.